client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Grid, Typography } from '@mui/material';
 import PageContainer from '../../components/PageContainer/PageContainer';
+import { io } from 'socket.io-client';
 
 export default function Dashboard(): JSX.Element {
+  useEffect(() => {
```

Comment:
Let's initialize the socket within the `SocketContext` after checking if a user is logged in 👍🏽